package.json

```
@@ -59,6 +59,8 @@
 		"@cloudflare/privacypass-ts": "0.5.1",
 		"@sentry/cli": "2.26.0",
 		"@sentry/types": "7.95.0",
+		"cron-parser": "^4.9.0",
```

```suggestion
	  "cron-parser": "4.9.0",
```
let's fix the dependency, similar to others